src/module/actor/actor-sheet.js

```diff
@@ -534,7 +534,17 @@ export class OseActorSheet extends ActorSheet {
     });
 
     html.find(".item-delete").click((event) => {
-      this._removeItemFromActor(event);
+      const item = this._getItemFromActor(event);
+
+      if (item.type !== "container" || !item?.system?.itemIds?.length > 0)
```

Comment:
`item.type` => `item?.type`, since we're checking on the second condition

Reply:
Indeed, this makes sense!